packages/apptive_grid_form/lib/src/translation/l10n/intl_de.json

```diff
@@ -20,5 +20,6 @@
     "attachmentSourceFiles": "Dateien auswählen",
     "searchUserNoResult": "Kein Nutzer mit \"$query\" gefunden",
     "progressProcessAttachment": "Anhänge werden verarbeitet [$processed/$total]",
-    "progressSubmitForm": "Formular einsenden"
+    "progressSubmitForm": "Formular einsenden",
+    "multilineTextOverflowError": "Ein mehrzeiliger Text wurde einem nicht mehrzeiligem Textfeld zugewiesen. Bitte passen sie das Formular an."
```

Comment:
Is this really an appropiate message? I would assume people seeing a form/this error might not necessarily have access to the form.

Reply:
removed